Minor changes to Version Control - Intro page and placeholder for Set-up section

MarkCBJSS opened the pull request:
Hi All,

I've done some minor edits on the Intro to Version Control page, nothing major. Just tweaks to grammar / sentence structure, highlights of links/buttons for look and feel consistency, added some sub-bullets to sections for context.

I also added a folder for a Set-up tutorial I'm working on. The thinking being we can split out the whole part about downloading, set-up, SSH keys, etc. then have the current two tutorials showing how to use Git / GitHub. Currently writing the Set-up tut now and I'll then modify the current pages to point Students to it.

MarkCBJSS committed:
Added folders and first files for Set-up tutorial

MarkCBJSS committed:
Minor changes - grammar, highlighted text naming buttons. Added sub-points on Why use version control section.

KimberleyCook commented on version-control/introduction/tutorial.md:
```diff
@@ -38,21 +38,23 @@ Wikipedia also holds a history of all changes.
 
-- When used on a regular basis, version control helps you to store your files safely. It makes it easy to figure out what broke your code as you can roll back to a previous version and work out when things last worked as expected.
+- When used on a regular basis, version control helps you to store your files safely. It makes it easy to figure out what broke your code, as you can roll back to a previous version and work out when things last worked as expected.
+    - With no version control in place you'll only have one copy of your file, then when it breaks there's no way to get back to good code!
```
rather than the phrase "good code" could we say "working code" instead please

MarkCBJSS replied:
Great - will change that phrase.

KimberleyCook commented on version-control/introduction/tutorial.md:
```diff
@@ -78,23 +80,25 @@ Some popular project using git:
-- commit: Applying any changes you have made to the **repository**
+- **Checkout**: When you retrieve code from a **repository**, to you local machine
+
+- **Commit**: Applying any changes you have made to the **repository**
```
Nice, this makes it a little clearer to read 💃

MarkCBJSS replied:
Just noticed a spelling error... will fix :]

KimberleyCook reviewed (commented):
I like the changes you've made here thank you very much. Could you remove the new .md  file from this PR please so that this PR stays strictly improving the existing one, and then I can merge these change in. 

Then do another PR adding the set up stage document. Would that be ok?

MarkCBJSS committed:
Deleted tutorial.md from set-up folder

Removing this focuses the commit just on tweaks to intro/tutorial.md file.
I'll add this back in one the re-write of this section is complete.

KimberleyCook reviewed (approved):
Thank you so much for this update, @MarkCBJSS
